packages/build/tests/plugins/snapshots/tests.js.md

```diff
@@ -15998,20 +15998,20 @@ Generated by [AVA](https://avajs.dev).
     ␊
     > Flags␊
       debug: true␊
-      repositoryRoot: packages/build/tests/plugins/fixtures/ts_transpile␊
+      repositoryRoot: /tmp-dir␊
       testOpts:␊
         pluginsListUrl: test␊
         silentLingeringProcesses: true␊
     ␊
     > Current directory␊
-      packages/build/tests/plugins/fixtures/ts_transpile␊
+      /tmp-dir␊
     ␊
     > Config file␊
-      packages/build/tests/plugins/fixtures/ts_transpile/netlify.toml␊
+      /tmp-dir/netlify.toml␊
     ␊
     > Resolved config␊
       build:␊
-        publish: packages/build/tests/plugins/fixtures/ts_transpile␊
+        publish: /tmp-dir␊
```

Comment:
This is due to now using `copyRoot` with this test. 

We use `copyRoot` to test that `@netlify/build` types are used even when not available in the project's `node_modules` directory.